react/ProductList.js

```diff
@@ -100,6 +102,13 @@ ProductList.getSchema = props => {
         default: ProductList.defaultProps.maxItems,
         isLayout: true,
       },
+      gap: {
+        title: 'editor.shelf.gap.title',
+        type: 'string',
+        enum: ['0x', '1x', '2x', '3x'],
```

Comment:
You should use your enum values here and not repeat the information of it. Create a enum file or export it.